spec/models/outgoing_message_spec.rb

```diff
@@ -536,40 +536,6 @@ class TestError < StandardError; end
             to include("Please enter your letter requesting information")
         end
 
-        context "when a censor rule changes the default text" do
```

Comment:
Introduced in https://github.com/mysociety/alaveteli/commit/7619291e4cb7531b6d71ee6dec2dae3818c5933d; looks like it's related to this part of the conversation https://github.com/mysociety/alaveteli/pull/3270#issuecomment-225127010.

Reply:
Notes for future me; still not convinced these specs are testing anything.

Why would the default letter need censor rules applying... for example with a censor rule doing `s/07912345678/00000000000/`, [this URL](http://0.0.0.0:3000/new/msw?default_letter=07912345678+alice@localhost.com) doesn't mask the number or email in the form textarea. Once the user previews the request the default letter isn't being used and the message body is like any other request - the number is redacted (by censor rule) and email masked as normal.